### PR TITLE
[11.0] [MIG] sale_order_line_sequence

### PR DESCRIPTION
**Pull request by remihb**

### CODE REVIEW

**Comment by tafaRU** [comment]
@remihb, thanks a lot for your PR!
Could you please take the opportunity to regenerate README file according to https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst ?

See https://github.com/OCA/web/pull/963#issuecomment-400344328 for furher info.

Thank you in advance.

**Comment by sebalix** [comment]
FYI, @remihb is under the ABF OSIELL CLA. Thank you!

File: sale_order_line_sequence/__init__.py
Comment (nikul-serpentcs) [inline sale_order_line_sequence/__init__.py]: @remihb Remove Copyright from all `__init__` file

File: sale_order_line_sequence/__manifest__.py
Comment (nikul-serpentcs) [inline sale_order_line_sequence/__manifest__.py]: @remihb IMO Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: sale_order_line_sequence/views/report_saleorder.xml
Comment (nikul-serpentcs) [inline sale_order_line_sequence/views/report_saleorder.xml]: here use `hasclass`

File: sale_order_line_sequence/views/report_saleorder.xml
Comment (nikul-serpentcs) [inline sale_order_line_sequence/views/report_saleorder.xml]: same here

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by remihb** [comment]
Ok, I think everything is fine by now.
I regenerate the README with the `oca-gen-addon-readme` command, so it's still a fake README but as far as I understand, there's a bot which is going to rewrite it later, isn't it?
Thanks for your help.

**Comment by tafaRU** [comment]
> there's a bot which is going to rewrite it later, isn't it?

No, if you regenerate it by yourself, as in your case!

**Comment by remihb** [comment]
Ok, I think I misunderstood the command.
So I regenerate the README according to the branch where the module will be merge in OCA.

**Review by sebalix — COMMENTED** [review]
@remihb LGTM, can you squash your migration commits?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional test.

**Comment by tafaRU** [comment]
@remihb, could you please extend the DESCRIPTION section adding that the sequence is also propagate to the sale order report?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Review by sebalix — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please attend remaining comments before merging.

**Comment by sbidoul** [comment]
@pedrobaeza if there are blocking changes, someone must make a "change requested" review to avoid the bot marking it approved / ready to merge.

**Comment by pedrobaeza** [comment]
Well, the workflow has been the contrary: I have seen the comment about merging, and then check that not everything has been honored.

**Comment by nikul-serpentcs** [comment]
@pedrobaeza @tafaRU Can you please review and move forward?

**Comment by pedrobaeza** [comment]
@nikul-serpentcs there are several comments from you that are not attended, isn't it?

**Comment by nikul-serpentcs** [comment]
Yes @pedrobaeza, But my comments are solved.
I think not any other comments are here

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs you can mark those comments as resolved so it is more clear. Thanks.

**Comment by pedrobaeza** [comment]
OK, merging.

**Pull request merged**
